interop/run.py

```diff
@@ -64,7 +65,7 @@ def _check_impl_is_compliant(self, name: str) -> bool:
     client_log_dir = tempfile.TemporaryDirectory(dir="/tmp", prefix="logs_client_")
 
     # check that the client is capable of returning UNSUPPORTED
-    logging.info("Checking compliance of %s client", name)
+    logging.debug("Checking compliance of %s client", name)
```

Comment:
I think this might be good to leave as info, otherwise there's a blank console. It's helpful to know what is going on.

Reply:
This change was necessary since I changed the log level in line 220. From the user's perspective, this shouldn't have any effect.